dev-util/packer/packer-1.5.6.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2019 Gentoo Authors
```

Comment:
You should update this to 1999-2020. You can commit with a tool called `repoman` that does this automatically for you.